src/lib/discord/scripts/registerCommands.ts

```diff
@@ -14,9 +14,7 @@ import fetch from 'node-fetch'
 async function registerCommands() {
 	const bulkUpdate: RESTPutAPIApplicationCommandsJSONBody = commands.map(c => c.json)
 
-	console.log(JSON.stringify(bulkUpdate, null, 2))
-
-	const json = await fetch(GLOBAL_COMMAND_API_URL, {
+	fetch(GLOBAL_COMMAND_API_URL, {
```

Comment:
This should still be awaited so it only says it registered the commands after the HTTP request was sent
```suggestion
	await fetch(GLOBAL_COMMAND_API_URL, {
```